.travis.yml

```diff
@@ -1,7 +1,16 @@
 language: python
 python:
     - "2.7"
+    - "3.4"
+    - "3.5"
+addons:
+    postgresql: "9.3"
```

Comment:
I couldn't remember what version we were targeting, but I assume we should work with any recent version.